packages/status-js/src/helpers/get-objects-difference.test.ts

```diff
@@ -0,0 +1,25 @@
+import { getObjectsDifference } from './get-objects-difference'
+
+describe('getObjectsDifference', () => {
+  it('returns correct difference', () => {
+    const oldObject = { a: 1, b: 2, c: 3 }
+    const newObject = { c: 3, d: 4, e: 5 }
+
+    expect(getObjectsDifference(oldObject, newObject)).toEqual({
+      added: {
+        d: 4,
+        e: 5,
+      },
+      removed: ['a', 'b'],
+    })
+  })
+
+  it('returns empty arrays for the same object', () => {
+    const object = { a: 1, b: 2, c: 3 }
+
+    expect(getObjectsDifference(object, object)).toEqual({
+      added: {},
+      removed: [],
```

Comment:
Non-blocking: I'd keep the same structure (`{}`) for both if a use of `removed` ever needed not just `keys` but `values` too.